submissions/kt-std/js-dom/script.js

```diff
@@ -0,0 +1,145 @@
+            'Earth is the third planet from the Sun and the only astronomical object known to harbor life. About 29% of Earth\'s surface is land consisting of continents and islands. The remaining 71% is covered with water, mostly by oceans but also lakes, rivers and other fresh water, which together constitute the hydrosphere.',
+        period: '365 Earth days',
+        position: '03',
+    },
+    {
+        heading: 'Mars',
+        paragraph:
+            'The fourth planet from the Sun and the second-smallest planet in the Solar System, being larger than only Mercury. Mars carries the name of the Roman god of war and is often referred to as the \'Red Planet\'. The days and seasons are comparable to those of Earth, because the rotational period as well as the tilt of the rotational axis relative to the ecliptic plane are similar.',
+        period: '687 Earth days',
+        position: '04',
+    },
+];
+
+document
+    .querySelector('#menu-btn')
+    .addEventListener('click', function display(e) {
+        menu.classList.toggle('visible');
+    });
+
+document
+    .querySelector('#menu')
+    .addEventListener('click', function display(e) {
+    const sectionName = e.target.innerHTML.toLowerCase();
+    index = CONTENT.findIndex((el) => el.heading.toLowerCase() === sectionName);
+    setData(index);
+});
+
+const addAnimation = (element, ...animationOptions) =>
+    element.animate(...animationOptions);
+
+const updateBackground = (element, path, id) => {
+    element.style.backgroundImage =
+        window.innerWidth < SCREEN_WIDTH
+            ? `url('${path + id + '_small'}.jpg')`
+            : `url('${path + id}.jpg')`;
+    addAnimation(
+        element,
+        [
+            {
+                opacity: 0.1,
+            },
+            {
+                opacity: 1,
+            },
+        ],
+        {
+            duration: 500,
+        }
+    );
+};
+
+const setInnerData = (element, data) => (element.innerHTML = data);
+
+const setData = (index) => {
+    const image = document.getElementById('tabImage'),
+        heading = document.querySelector('.tab__header'),
+        headerMenu = document.querySelector('.header'),
+        container = document.querySelector('.container'),
+        paragraph = document.querySelector('.tab__paragraph'),
+        period = document.querySelector('.statistics__text'),
+        position = document.querySelector('#position');
+
+    updateBackground(image, ASSETS_PATH, CONTENT[index].heading.toLowerCase());
+    setInnerData(heading, CONTENT[index].heading);
+    setInnerData(position, CONTENT[index].position);
+    setInnerData(paragraph, CONTENT[index].paragraph);
+    addAnimation(
+        container,
+        [
+            {
+                opacity: 0,
+            },
+            {
+                opacity: 1,
+            },
+        ],
+        {
+            duration: 600,
+        }
+    );
+
+    setInnerData(period, CONTENT[index].period);
+    if (window.innerWidth > 800) {
```

Comment:
Please use the `SCREEN_WIDTH` variable here

Reply:
Sorry, missed that one previous time